### DISCUSSION TITLE
Planning continuous paths instead of Waypoints

### DISCUSSION THREAD

**Discussion by parthpatel0** [stronnag/mwptools#142] [General]
Hi All,

I was wondering if it's possible to plan continuous paths instead of using just waypoints in mwptool ? 
My application requires the quad to fly in a circle and I imagine using just waypoints won't suffice. Do you have any ideas or suggestions for doing so ?

Thanks

**Comment by stronnag** [comment]
The FC can only fly points between fixed WPs, so that's all MWP offers. You can use the "Add Shape" option to make polygons approximating to circles, see the [user guide](https://stronnag.github.io/mwptools/mission-editor/#add-shape) and [referenced tutorial video](https://stronnag.github.io/mwptools/mission-editor/#overview).
